### PR TITLE
Revalidation 2022 - Chinese (Traditional and Simplified)

### PR DESCRIPTION
**Pull request by SooluThomas**
### Summary

Annual revalidation 2022 for Chinese (Traditional and Simplified)

### TODO

- [x] the members with **NO ✅** will be removed **from the list and crowdin** after the TL(s) @coruscating @wslu42 reviews this PR
- [x] Remove the users from list
- [x] remove ✅ to confirm the new list of contributors
- [ ] remove the users from Crowdin

### CODE REVIEW

**Comment by wslu42** [comment]
Sorry for the late reply, here are the list of names that need to be added/updated manually as they will still be actively translating the works:

**Chinese-Simplified (ZH-CN)**
Wen-Sen Lu （TL)
Junye Huang
Yikai Wang
@@ -446,14 +446,14 @@ Yu-Heng, Lin

**Chinese-Traditional (ZH-TW)**
Junye Huang
Ran-Yu Chang
Pei-Che Hsu
HSU,YU-CHAO
Yikai Wang
Ruiqi Zhang

note: I'm still asking the rest of participants and will be updating the list by the end of today (4/19)

File: LOCALIZATION_CONTRIBUTORS [hunk 1]
```diff
@@ -426,18 +426,18 @@ Felix makungu mkhaya
 
 Chinese-Simplified (ZH-CN)
 
-Helena Zhang (TL)
+Helena Zhang (TL) ✅
 Wen-Sen Lu （TL)
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:430]: Wen-Sen Lu （TL) ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 2]
```diff
@@ -426,18 +426,18 @@ Felix makungu mkhaya
...
 Junye Huang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:431]: Junye Huang ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 3]
```diff
@@ -426,18 +426,18 @@ Felix makungu mkhaya
...
-Lebin Yu
+Lebin Yu ✅
 Jia Qing Tan
 Xueyan Niu
-Xiaoyuan Liu
+Xiaoyuan Liu ✅
 Yikai Wang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:436]: Yikai Wang ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 4]
```diff
@@ -446,14 +446,14 @@ Yu-Heng, Lin
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:446]: Yu-Heng, Lin ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 5]
```diff
@@ -446,14 +446,14 @@ Yu-Heng, Lin
 
 Chinese-Traditional (ZH-TW)
 
-Helena Zhang (TL)
-Wen-Sen Lu (TL)
+Helena Zhang (TL) ✅
+Wen-Sen Lu (TL) ✅
 Junye Huang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:451]: Junye Huang ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 6]
```diff
@@ -446,14 +446,14 @@ Yu-Heng, Lin
...
-Lebin Yu
+Lebin Yu ✅
 Ran-Yu Chang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:453]: Ran-Yu Chang ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 7]
```diff
@@ -446,14 +446,14 @@ Yu-Heng, Lin
...
-Alan Leung
-Yu-Cheng Lin
-yi-kai Lee
+Alan Leung ✅
+Yu-Cheng Lin ✅
+yi-kai Lee ✅
 Jia Qing Tan
 Pei-Che Hsu
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:458]: Pei-Che Hsu ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 8]
```diff
@@ -446,14 +446,14 @@ Yu-Heng, Lin
...
 HSU,YU-CHAO
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:459]: HSU,YU-CHAO ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 9]
```diff
@@ -472,9 +472,9 @@ Yu-ping-Lin
 YU-JUI,LIN
 chau Kwai Yin Jodie
 Hsuan-Yu CHOU
-Chih-Han (Robin) Huang
+Chih-Han (Robin) Huang ✅
 Chang Li-lin
-Xiaoyuan Liu
+Xiaoyuan Liu ✅
 Yikai Wang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:478]: Yikai Wang ✅

File: LOCALIZATION_CONTRIBUTORS [hunk 10]
```diff
@@ -472,9 +472,9 @@ Yu-ping-Lin
...
 Ruiqi Zhang
```
Comment (wslu42) [inline LOCALIZATION_CONTRIBUTORS:479]: Ruiqi Zhang ✅

**Comment by SooluThomas** [comment]
Thank you @wslu42. I have updated the list based on your above comments. Please let me know if that's it or if you're waiting on responses from the contributors.

Feel free to simply add a comment with all names that need a ✅. 😄

**Comment by wslu42** [comment]
@SooluThomas I think we can deem the previous list as final and close this case, thanks!

**Review by wslu42 — APPROVED** [review]

**Comment by SooluThomas** [comment]
@wslu42 Please confirm the list one last time 😊

**Review by wslu42 — APPROVED** [review]
Confirming the changes in the list, thanks!

**Pull request merged**